apps/web/components/ServiceInformation/ServiceInformation.tsx

```diff
@@ -1,117 +1,281 @@
 
+  // TODO When enviroment is chosen set the version options, default is set to newest version.
+  const enviromentOptions: Array<SelectOption> = service.environments.map(
+    (x) => {
+      return {
+        label: x.environment,
+        value: x.environment,
+      }
+    },
+  )
+  const [selectedEnviromentOption, setSelectedEnviromentOption] = useState<
+    SelectOption
+  >(enviromentOptions[0])
+
+  const versionOptions: Array<SelectOption> = service
+    ? service.environments[0].details.map((x) => {
+        // TODO: Change this when we add environmental aware services
+        return {
+          label: x.version.split('-').pop(),
+          value: x.xroadIdentifier,
+        }
+      })
+    : [
+        {
+          label: n('noVersion'),
+          value: {
+            instance: '',
+            memberClass: '',
+            memberCode: '',
+            serviceCode: '',
+            subsystemCode: '',
+          },
+        },
+      ]
+
+  const [selectedVersionOption, setSelectedVersionOption] = useState<
+    SelectOption
+  >(versionOptions[0])
+
+  const [serviceDetail, setServiceDetail] = useState<ServiceDetail>(
+    service.environments[0].details[0],
+  )
+
+  const onSelectVersion = (versionOption: SelectOption) => {
+    const tempServiceDetail = service.environments
+      .find((e) => e.environment === selectedEnviromentOption.value)
+      .details.find((e) => e.xroadIdentifier === versionOption.value)
+
+    setServiceDetail(tempServiceDetail)
+    setSelectedVersionOption(versionOption)
+    onSelectChange(tempServiceDetail)
+  }
+
+  const onSelectEnviroment = (enviromentOption: SelectOption) => {
+    //TODO this function needs to be finished similar to onSelectVersion
+    setServiceDetail(
+      service.environments
+        .find((e) => e.environment === enviromentOption.value)
+        .details.find(selectedVersionOption.value),
+    )
+    setSelectedEnviromentOption(enviromentOption)
+  }
+
   return (
     <Box>
-      <Inline space={1}>
-        <Text variant="h1" as="h1">
-          {service.title}
+      <Box marginTop={1} marginBottom={3}>
+        <Box display="flex" alignItems="flexStart">
+          <Text variant="h1" as="h1">
+            {service.title}
+          </Text>
+          {service.pricing.length > 0 && (
+            <Box marginLeft={1}>
+              <Tag>{n(`pricing${capitalize(service.pricing[0])}`)}</Tag>
+            </Box>
+          )}
+        </Box>
+        <Text variant="intro" paddingTop={2}>
+          {service.description}
         </Text>
-        {service.pricing.length > 0 && (
-          <Box>
-            <Tag variant="white" outlined>
-              {n(`pricing${capitalize(service.pricing[0])}`)}
-            </Tag>
+      </Box>
+
+      <Box paddingBottom={3}>
+        <Divider />
+      </Box>
+
+      <Box
+        display="flex"
+        flexDirection={['column', 'row']}
+        justifyContent={['flexStart', 'spaceBetween']}
```

Comment:
Wondering if both `xs` and `sm` breakpoints should be `column` and `flexStart` for the mobile desing, then desktop `md` and above.